### PR TITLE
add custom domain to koumakan.eth

### PR DESCRIPTION
**Pull request by u-u-z**
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- ~Add images for your space~ (deprecated) Add the image URL on the space settings directly
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -378,5 +378,6 @@
   "voting-v2.multiplier.finance": "v2-multiplierfinance.eth",
   "vote.mgh.dao": "metagamehub.eth",
   "gov.maidcoin.org": "maidcoin.eth",
-  "vote.cyberkongz.com": "cyberkongz.eth"
+  "vote.cyberkongz.com": "cyberkongz.eth",
+  "dao.baka.cool":"koumakan.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:382]: Please add a space here.
```suggestion
  "dao.baka.cool": "koumakan.eth"
```

Comment (u-u-z) [reply spaces/domains.json]: resolve

File: spaces/domains.json [hunk 2]
```diff
@@ -381,5 +381,6 @@
   "vote.mgh.dao": "metagamehub.eth",
   "gov.maidcoin.org": "maidcoin.eth",
   "vote.cyberkongz.com": "cyberkongz.eth",
+  "dao.baka.cool": "koumakan.eth"
   "governance.yfdai.finance": "yfdaifinance.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:385]: A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dao.baka.cool": "koumakan.eth",
  "governance.yfdai.finance": "yfdaifinance.eth"
```

**Comment by u-u-z** [comment]
Im so sorry .

**Pull request closed without merging**